tests/integration/test_openapi.py

```diff
@@ -49,3 +50,23 @@ def test_openapi_endpoint_head_method():
     response = client.head("/openapi.json")
     assert response.status_code == requests.codes.ok
     assert response.text == ""
+
+
+def test_openapi_content():
+    """Check if the pre-generated OpenAPI schema is up-to date."""
+    # retrieve pre-generated OpenAPI schema
+    with open("docs/openapi.json", "r") as fin:
```

Comment:
What is `fin`?

Reply:
input file, fout=output file, idioms

Reply:
ok so it is file in/file out

Reply:
yup, you can see it in sources (instead of plain `f`) like https://stackoverflow.com/questions/8384384/python-concise-elegant-way-to-reformat-a-set-of-text-files